third_party/VENDOR-LICENSE/gopkg.in/yaml.v3/NOTICE

```diff
@@ -1,5 +1,4 @@
-/*
-Copyright 2019 The Knative Authors.
+Copyright 2011-2016 Canonical Ltd.
```

Comment:
@maximilien What is this new copyright?

Reply:
Interesting...  I am guessing that Canonical is contributing to the Golang YAML parser and / or took over it.

Since this is third party, let me tag @evankanderson who might have seen or know how to deal with this. Thx.

Reply:
This looks like Git was too clever in move detection... this is detecting a deleted file from `pkg/reconciler/testing/clock.go` as a _move_ to a `NOTICE` file.

Reply:
I think it's fine. Interesting to note but the Apache license and MIT are there